test/boost/sstable_3_x_test.cc

```diff
@@ -5197,8 +5197,7 @@ static void test_sstable_write_large_row_f(schema_ptr s, reader_permit permit, r
     sstables::test_env::do_with_async([&] (auto& env) {
         test_db_config.host_id = locator::host_id::create_random_id();
         tmpdir dir;
-        auto sst = env.manager().make_sstable(
-                s, dir.path().string(), generation_from_value(1), version, sstables::sstable::format_types::big);
+        auto sst = env.make_sstable(s, dir.path().string(), 1, version, sstables::sstable::format_types::big);
 
```

Comment:
make_sstables() needs to be adjusted for using generation_type. with a look forward to generation type being abstracted.

Reply:
I am not saying to do it now, though. just that it needs to be done :-)